controllers/utils/recorder/common.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package recorder
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Applied struct {
+	Id string
+}
+
+func (a Applied) Type() string {
+	return "Normal"
+}
+
+func (a Applied) Reason() string {
+	return "Applied"
+}
+
+func (a Applied) Message() string {
+	return fmt.Sprintf("Successfully apply chaos for %s", a.Id)
+}
+
+func (a Applied) Parse(message string) ChaosEvent {
```

Comment:
I think we don't need these `Parse` anymore, right?

Reply:
Yes. It seems that I left some `Parse`... They are removed now :beers: